app/src/controllers/itemsHistory/reducer.js

```diff
@@ -22,19 +22,23 @@ const visibleItemsCountReducer = (state = 0, { type, payload }) => {
   }
 };
 
-const historyReducer = (namespace) => (state = [], { type, payload, meta }) => {
-  if (meta && meta.namespace && meta.namespace !== namespace) {
+const historyReducer = (state = [], { type, payload, meta }) => {
+  if (meta && meta.namespace && meta.namespace !== NAMESPACE) {
     return state;
   }
   let newItemsHistory;
+  let reversedPayload;
   switch (type) {
     case FETCH_SUCCESS:
       if (state.length === 0) {
-        return payload;
+        return payload.reverse();
       }
+      reversedPayload = payload.reverse();
```

Comment:
`const reversedPayload = ...`

Reply:
We couldn't define variables in switch block